core-domain/src/main/java/org/uniprot/core/unirule/impl/FtagConditionValueImpl.java

```diff
@@ -0,0 +1,39 @@
+package org.uniprot.core.unirule.impl;
+
+import java.util.Objects;
+
+import org.uniprot.core.impl.ValueImpl;
+import org.uniprot.core.unirule.FtagConditionValue;
+
+public class FtagConditionValueImpl extends ValueImpl implements FtagConditionValue {
```

Comment:
See comment above about class name involving F. If not applicable, please ignore.